buildbot/osuosl/master/config/builders.py

```diff
@@ -2526,21 +2526,30 @@
     'tags'  : ['flang'],
     'workernames' : ["linaro-flang-aarch64-libcxx"],
     'builddir': "flang-aarch64-libcxx",
-    'factory' : UnifiedTreeBuilder.getCmakeWithNinjaBuildFactory(
-                    clean=True,
-                    checks=['check-flang','check-flang-rt'],
-                    depends_on_projects=['llvm','mlir','clang','flang','flang-rt'],
-                    extra_configure_args=[
+    'factory' : FlangBuilder.getFlangOutOfTreeBuildFactory(
+                    checks=['check-flang'],
+                    llvm_extra_configure_args=[
                         "-DLLVM_TARGETS_TO_BUILD=AArch64",
-                        "-DLLVM_INSTALL_UTILS=ON",
                         "-DCMAKE_CXX_STANDARD=17",
                         "-DLLVM_ENABLE_WERROR=OFF",
+                        "-DBUILD_SHARED_LIBS=ON",
```

Comment:
Are we enabling shared libs because the previous configuration did so, or do we actually need it?

I could understand needing it for llvm so that the subsequent flang build can use them, but further into the build it does not seem necessary.

(I don't care about how long it takes, just about being as close as we can to the defaults)

Reply:
Shared libs are being enabled just because the previous configuration did so. It's probably not needed.

If no errors occur, do you think it would be better to avoid shared libs in all builds (llvm, flang, flang-rt)?

Reply:
Maybe we did this as some way to save build time? I'm not sure if it actually does though.

Yes let's avoid them here unless the alternative has errors or lots more configuration changes. (any other bot can stay as it is)